gammapy/makers/background/tests/test_phase.py

```diff
@@ -95,3 +99,127 @@ def test_check_phase_intervals(pars):
     assert_allclose(
         PhaseBackgroundMaker._check_intervals(pars["p_in"]), pars["p_out"], rtol=1e-5
     )
+
+
+@pytest.mark.parametrize(
+    "pars",
+    [
+        {
+            "p_in": ["5029747", "point-like"],
+            "p_out": [
+                np.reshape(
+                    np.array(
+                        [
+                            58,
+                            19,
+                            2,
+                            2,
+                            0,
+                            0,
+                        ]
+                    ),
+                    (6, 1, 1),
+                ),
+                np.reshape(np.array([163, 46, 15, 1, 0, 0]), (6, 1, 1)),
+            ],
+        },
+        {
+            "p_in": ["5029747", "not-point-like"],
+            "p_out": [
+                np.reshape(
+                    np.array(
+                        [
+                            23,
+                            18,
+                            2,
+                            2,
+                            0,
+                            0,
+                        ]
+                    ),
+                    (6, 1, 1),
+                ),
+                np.reshape(
+                    np.array(
+                        [
+                            51,
+                            35,
+                            15,
+                            1,
+                            0,
+                            0,
+                        ]
+                    ),
+                    (6, 1, 1),
+                ),
+            ],
+        },
+        {
+            "p_in": ["111630", "not-point-like"],
+            "p_out": [
+                np.reshape(
+                    np.array(
+                        [
+                            15,
+                            5,
+                            5,
+                            8,
+                            2,
+                            1,
+                        ]
+                    ),
+                    (6, 1, 1),
+                ),
+                np.reshape(
+                    np.array(
+                        [
+                            24,
+                            13,
+                            15,
+                            12,
+                            2,
+                            0,
+                        ]
+                    ),
+                    (6, 1, 1),
+                ),
+            ],
+        },
+    ],
+)
+@requires_data()
+def test_make_counts(observations, phase_bkg_maker, pars):
+
+    maker = SpectrumDatasetMaker(
+        containment_correction=False, selection=["counts", "exposure", "edisp"]
+    )
+
+    e_reco = MapAxis.from_energy_bounds(0.05, 100, nbin=6, unit="TeV", name="energy")
+    e_true = MapAxis.from_energy_bounds(
+        0.01, 300, nbin=15, unit="TeV", name="energy_true"
+    )
+
+    if pars["p_in"][0] == "111630":
```

Comment:
Maybe you can use the HESS DL3 Crab runs to get rid of this if else block

Reply:
I think I can't because I need a rad_max to test the feature and this is not in the HESS DL3 gammapy data. 
Maybe I'm mistaken on that

Reply:
Full enclosure IRFs should not have a `rad_max`, no? This tests the "standard analysis" case, which should be equivalent for the hess and cta datasets I think

Reply:
But this also test that the rad_max is taken into account in the making of counts, so we need a run with rad_max to test that, hence the MAGIC run right?